docs/sitemap.md

```diff
@@ -0,0 +1,22 @@
+# Sitemap
+
+The `/sitemap.xml` URL provides the complete BigCommerce sitemap, which includes canonical URLs for all channels associated with Catalyst.
```

Comment:
What domain does the sitemap have?

Reply:
The top level sitemap index has the canonical URLs, but the individual sitemaps have the site URL.

## Sitemap Index
![Screenshot 2024-08-29 at 13 56 22](https://github.com/user-attachments/assets/20548aee-e71d-439f-8f91-a346d3338d80)

## Individual Sitemap
![Screenshot 2024-08-29 at 13 56 56](https://github.com/user-attachments/assets/9b8eb938-b15f-4d96-a94a-8b80d74aa960)